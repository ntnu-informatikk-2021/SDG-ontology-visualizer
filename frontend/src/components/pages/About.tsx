import React from 'react';

const About = () => <div>test1</div>;

export default About;
