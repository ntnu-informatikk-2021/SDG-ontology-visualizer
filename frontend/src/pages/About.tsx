import React from 'react';

const About = () => {
  const test = { test1: String };
  console.log(test);

  return <div>test1</div>;
};

export default About;
