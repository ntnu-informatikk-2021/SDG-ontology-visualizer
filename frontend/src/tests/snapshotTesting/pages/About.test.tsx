import React from 'react';
import renderer from 'react-test-renderer';
import About from '../../../components/pages/About';

it('renders when there are no items', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
