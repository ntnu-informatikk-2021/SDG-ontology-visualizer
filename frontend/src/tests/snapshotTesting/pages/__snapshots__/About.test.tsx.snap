// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there are no items 1`] = `
<div
  className="css-awrlxj"
>
  <div
    className="chakra-stack css-14nowqt"
  >
    <h2
      className="chakra-heading css-0"
    >
      Om prosjektet
    </h2>
    <p
      className="chakra-text css-1ezsviu"
    >
      Dette er en prototype laget for Trondheim kommune i forbindelse med bacheloroppgave i informatikk. Applikasjonen gir mulighet til å utforske bærekraftsmålene og finne sammenhenger og relasjoner til ulike perspektiver satt av Trondheim kommune. Dataen i applikasjonen er lagret og hentet fra en ontologi, som også definerer relasjonene mellom disse.
    </p>
    <a
      className="chakra-link css-otktsc"
      href="https://github.com/ntnu-informatikk-2021/SDG-ontology-visualizer"
      rel="noopener noreferrer"
      target="_blank"
    >
      <svg
        fill="currentColor"
        height="40"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 16 16"
        width="40"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"
          fillRule="evenodd"
        />
      </svg>
    </a>
  </div>
</div>
`;
